FirstTableViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  FirstTableViewController.swift
+//  ModelModel
+//
+//  Created by Stanislav Cherkasov on 12.12.17.
+//  Copyright © 2017 Stanislav Cherkasov. All rights reserved.
+//
+
+import UIKit
+
+class FirstTableViewController: UITableViewController {
+    
+    var usersArray = [User]()
+  
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = "USERS"
+        makeUser()
+    }
+    
+    func makeUser() {
+        usersArray.append(User(uname: "Vasiliy", uage: 17, uweight: 56, uheight: 166.8))
+        usersArray.append(User(uname: "Katya", uage: 22, uweight: 50.5, uheight: 170))
+        usersArray.append(User(uname: "Olga", uage: 30, uweight: 58.8, uheight: 160))
+        usersArray.append(User(uname: "Sveta", uage: 10, uweight: 35, uheight: 137))
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return usersArray.count
+    }
+
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cellFirst", for: indexPath)
+        
+        cell.textLabel?.text = usersArray[indexPath.row].name
```

Comment:
too bad, what if you will need all User data, and not just name? You should init Cell with a selected User Model, and Cell will decide how to fill itself on its own in its own init method, or e.g.` func fillWith(user: User)` method